codegenerator/cli/src/cli_args/interactive_init/mod.rs

```diff
@@ -118,8 +118,9 @@ pub async fn prompt_missing_init_args(
         None => {
             // TODO: input validation for name
             Text::new("Name your indexer:")
-                .with_default("My Envio Indexer")
+                .with_default("my_envio_indexer")
```

Comment:
Changed the default since it was an invalid name

Reply:
I like dashes for package names :)